src/ClickEncoder.cpp

```diff
@@ -176,8 +168,8 @@ void ClickEncoder::service(void)
     }
 
     if (doubleClickTicks > 0) {
-      doubleClickTicks--;
```

Comment:
Old and new code is buggy when doubleClickTicks is odd. Also varible is unsigned so never < 0. A different test must be done I think

Reply:
@technik-gegg what is correct code here? I think it should be just one dec? Right?

Reply:
I'd say the there's only one decrement needed, since all this code is meant to trigger for the 600ms, after which a **single click** is generated if there was no 2nd click in the meantime. Not sure why the author decided to add a second decrement here, which effectively reduces the trigger time to 300ms.